### DISCUSSION TITLE
The latest update broke links to the spaces with variants

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#664] [0. General bugs and questions] [answered]
Hello everyone!

The latest GitBook update (the one that includes "doc sites") broke links to all our documentation that uses variants. Not sure if anyone else is affected, but the issue is very frustrating for us because it targets our most popular docs and is seemingly unfixable from our end.  

How can we solve this issue as fast as possible? Thank you!

**Comment by aikoii** [comment]
Thanks for flagging @a-asharapova! We're looking into it 🙏

**Comment by emmerich** [comment, marked as answer]
Hey @a-asharapova , this one should be fixed now. It was caused by something we shipped last Friday. Please let us know if you still see the problem 🙏
